### PR TITLE
Add author add game classes

### LINKED ISSUE #16: [1pt] Create a Game class in a separate .rb file.

### LINKED ISSUE #19: [ 2pt ] All Game class properties visible in the diagram should be defined and set up in the constructor method.
- All Game class properties visible in the diagram should be defined and set up in the constructor method.

### LINKED ISSUE #20: [ 1pt ]All Author class properties visible in the diagram should be defined and set up in the constructor method
- All Author class properties visible in the diagram should be 
 defined and set up in the constructor method

### PR DESCRIPTION
**Pull request by mwenyoa**
### In this PR, i did the following:

- Create a Game class in a separate .rb file.
- All Game class properties visible in the diagram should be defined and set up in the constructor method.
- All Author class properties visible in the diagram should be defined and set up in the constructor method.

closes #16 
closes #19
closes #20

### CODE REVIEW

File: classes/author.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require_relative './app'
```
Comment (aimalamiri) [inline classes/author.rb]: This app class is included but never used, please remove it.

**Review by aimalamiri — CHANGES_REQUESTED** [review]
## Wow! You're so closed
### Status: requires changes

The project is almost completed although there are some minor changes that need to be investigated. please make sure to check them in the comments below.

File: classes/game.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Game
```
Comment (aimalamiri) [inline classes/game.rb]: According to the diagram, this class is a subclass of the Item class, so this class should be inherited from the Item class.

Comment (mwenyoa) [reply classes/game.rb]: Hi @aimalamiri, I have worked on the two issues, kindly find time to review my updated code.

**Review by aimalamiri — COMMENTED** [review]
## Congratulations
## Status: Approved

There is nothing to say rather than it's time to merge it.

Good Job @mwenyoa you made it finally.

**Review by aimalamiri — APPROVED** [review]

**Pull request merged**